apps/test-app/frontend/src/components/viewport/ViewportContentControl.tsx

```diff
@@ -27,11 +28,21 @@ export default function ViewportContentComponent(props: ViewportContentComponent
     setPrevIModel(props.imodel);
   }
   useEffect(() => {
+    const schemas = new SchemaContext();
+    schemas.addLocater(new ECSchemaRpcLocater(props.imodel.getRpcProps()));
```

Comment:
I think this should be moved to `MyAppFrontend` and shared across components (e.g. with stateless tree)

Reply:
Turns out `MyAppFrontend` already has a function for it, so switched to using it here and in stateless tree.